Agents/DashboardAgent/agent/src/main/java/uk/ac/cam/cares/jps/agent/dashboard/DashboardAgent.java

```diff
@@ -0,0 +1,189 @@
+package uk.ac.cam.cares.jps.agent.dashboard;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.json.JSONObject;
+import uk.ac.cam.cares.jps.agent.dashboard.json.DashboardClient;
+import uk.ac.cam.cares.jps.agent.dashboard.stack.StackClient;
+import uk.ac.cam.cares.jps.base.agent.JPSAgent;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * This class acts as the entry point of the compiled war.
+ *
+ * @author qhouyee
+ */
+@WebServlet(urlPatterns = {"/reset", "/status", "/setup"})
+public class DashboardAgent extends JPSAgent {
+    private static final Logger LOGGER = LogManager.getLogger(DashboardAgent.class);
+    // Agent starts off in valid state, and will be invalid when running into exceptions
+    private static boolean VALID = true;
```

Comment:
The Java naming convention is that only `static final` variables (global constants) are all caps.
It is more correct to initialise `valid` to `false` as it isn't valid until `init` has been run.
```suggestion
    // Agent starts off in invalid state, and will become valid when initialised without exceptions
    private static boolean valid = false;
```